ui/appui-layout-react/src/appui-layout-react/state/NineZoneStateReducer.ts

```diff
@@ -668,6 +668,11 @@ export function NineZoneStateReducer(
       const nzBounds = Rectangle.createFromSize(state.size);
       const containedBounds = preferredBounds.containIn(nzBounds);
 
+      const userSized =
+        tab.userSized ||
+        (tab.isFloatingWidgetResizable &&
+          /* istanbul ignore next */ !!tab.preferredFloatingWidgetSize);
```

Comment:
This gives the impression that if the widget is *not* resizable we will not take into account the `canFloat.defaultSize`, is that correct ? (It is even more important if the widget cannot be resized that the size given by the dev should be enforced)

Reply:
I don't believe so, it should take into account the canFloat.defaultSize. It is setting the userSized to true if the tab has it set as true or if the tab is resizable and is no longer its preferred size. The userSized is always set in our code like this as well. 
![image](https://github.com/iTwin/appui/assets/70602409/a13acb7f-c976-4df3-8b99-29b43cb698e8)

Reply:
Only issue I see is that if you "SendBack", then the position is lost and it resets without reusing the default position (or the last position), however it keeps the default size. But that is doing the same in master and do not appear to be related to this PR, should we file a different issue or you think this can be related to the area you already explored with this ?

https://github.com/iTwin/appui/assets/1904889/d6838930-ae6e-46cc-b417-ec318e442829

Reply:
That shouldn't be too bad to fix, I think I know what needs to be done. I'll just tack it onto this PR, it's in the same area I'm working on.